src/o365/spo/commands/tenant/tenant-recyclebinitem-remove.ts

```diff
@@ -0,0 +1,176 @@
+import request from '../../../../request';
+import commands from '../../commands';
+import SpoCommand from '../../../base/SpoCommand';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption, CommandCancel } from '../../../../Command';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  wait: boolean;
+  confirm?: boolean;
+}
+
+interface RestSpoOperation {
+  HasTimedout: boolean;
+  IsComplete: boolean;
+  PollingInterval: number;
+}
+
+class SpoTenantRecycleBinItemRemoveCommand extends SpoCommand {
+  private timeout?: NodeJS.Timer;
+
+  public get name (): string {
+    return commands.TENANT_RECYCLEBINITEM_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes the specified deleted Site Collection from Tenant Recycle Bin';
```

Comment:
We should change `Site Collection` to `site collection` and `Tenant Recycle Bin` to `tenant recycle bin`